frontend/src/NavigationPath.tsx

```diff
@@ -151,12 +151,19 @@ export enum NavigationPath {
   createPolicyAutomation = '/multicloud/governance/policyautomation/create/:namespace/:name',
   editPolicyAutomation = '/multicloud/governance/policyautomation/edit/:namespace/:name',
   policyDetails = '/multicloud/governance/policies/details/:namespace/:name',
+  discoveredByCluster = '/multicloud/governance/discovered/:apiGroup?/:apiVersion/:kind/:policyName/:policyNamespace',
```

Comment:
Actually, since we can't control the API version returned by search's index, maybe we can omit that. Also `useFetchPolicies` seems to require the `apiGroup` so we can make that a required field.